src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/AppBase.cs

```diff
@@ -426,5 +465,121 @@ private List<string> GetOptionIdsFromFormLayout(string formLayout)
 
             return dictionary;
         }
+
+        private async Task SendInstanceData(Instance instance)
+        {
+            Guid instanceGuid = Guid.Parse(instance.Id.Split("/")[1]);
+            int instanceOwnerPartyId = int.Parse(instance.InstanceOwner.PartyId);
+
+            foreach (DataElement dataElement in instance.Data)
+            {
+                if (!_appMetadata.EFormidling.DataTypes.Contains(dataElement.DataType))
+                {
+                    continue;
+                }
+
+                bool appLogic = _appMetadata.DataTypes.Any(d => d.Id == dataElement.DataType && d.AppLogic != null);
+
+                string fileName = appLogic ? $"{dataElement.DataType}.xml" : dataElement.Filename;
+                Stream stream = await _dataService.GetBinaryData(instance.Org, instance.AppId, instanceOwnerPartyId, instanceGuid, new Guid(dataElement.Id));
+
+                using (stream)
+                {
+                    bool successful = await _eFormidlingClient.UploadAttachment(stream, instanceGuid.ToString(), fileName);
+
+                    if (!successful)
+                    {
+                        _logger.LogError("// AppBase // SendInstanceData // DataElement {DataElementId} was not sent with shipment for instance {InstanceId} failed.", dataElement.Id, instance.Id);
+                    }
+                }
+            }
+        }
+
+        private async Task<StandardBusinessDocument> ConstructStandardBusinessDocument(string instanceGuid, Instance instance)
+        {
+            DateTime completedTime = DateTime.Now;
+
+            Sender digdirSender = new Sender
+            {
+                Identifier = new Identifier
+                {
+                    // 0192 prefix for all Norwegian organisations.
+                    Value = $"0192:{_appSettings.EFormidlingSender}",
```

Comment:
Gitt at vi har de andre settingsene i samme fil. Hva var argumentet for å ha akkurat denne settingen utenfor. Husker vi snakket om at det var lurt, men jeg er nå usikker på verdien

Reply:
Sender er det vi som styrer. Sender vil jo være Altinn / Digdir. Så det bør ikke være noe som apputvikler skal måtte konfigurere er vel det vi har tenkt.